azurerm/resource_arm_sql_server.go

```diff
@@ -74,6 +75,33 @@ func resourceArmSqlServer() *schema.Resource {
 				Computed: true,
 			},
 
+			"identity": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"type": {
+							Type:             schema.TypeString,
+							Required:         true,
+							DiffSuppressFunc: suppress.CaseDifference,
+							ValidateFunc: validation.StringInSlice([]string{
+								"SystemAssigned",
+							}, true),
```

Comment:
(as above) we're making all new fields case-sensitive - as such can we make this:

```suggestion
							}, false),
```
```suggestion
							}, true),
```

Reply:
Set this to `false`.